### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by kevinccbsg**
### Type:
* [ ] cicd: helm, docker & cicd adjustments.
* [X] feature: new capabilities.
* [X] fix: bug, hotfix, etc.
* [ ] refactor: enhacements.
* [ ] style: changes in styles.
* [ ] other: docs, tests.

### What's the focus of this PR:
This PR includes:

- Update docker node version to the LTS
- Update committing dependency
- Fix test file errors

### CODE REVIEW

File: generators/app/templates/root/_package.json [hunk 1]
```diff
@@ -8,12 +8,11 @@
     "start": "node index.js",
     "test:report": "cross-env NODE_ENV=test jest --detectOpenHandles --forceExit --coverage",
     "test": "cross-env NODE_ENV=test jest --detectOpenHandles --forceExit --reporters=default",
-    "lint": "eslint .",
-    "manifest": "node_modules/make-manifest/bin/make-manifest",
-    "postinstall": "node_modules/.bin/eslint . --fix"
+    "lint": "cross-env eslint .",
```
Comment (Betisman) [inline generators/app/templates/root/_package.json:11]: Is this needed?

File: generators/app/templates/test/.eslintrc [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+  },
+  "plugins": ["jest"],
+  "rules": {
+    "jest/no-disabled-tests": "error",
+    "jest/no-focused-tests": "error",
+    "jest/no-identical-title": "error",
+    "jest/prefer-to-have-length": "warn",
+    "jest/valid-expect": "error"
+  }
+}
```
Comment (Betisman) [inline generators/app/templates/test/.eslintrc:13]:
```suggestion
}

```

File: generators/app/templates/root/_system.js [hunk 3]
```diff
@@ -3,9 +3,9 @@ const initMetrics = require('systemic-azure-metrics');
 const { join } = require('path');
 
 module.exports = () =>
-  new System({ name: '<%= name %>' })
-  // metrics need initialising before systems related to connections start
-  .add('metrics', initMetrics())
-  .dependsOn('config')
-  .bootstrap(join(__dirname, 'components'));
+  new System({ name: 'poc-generator' })
```
[the remark this answers is not in the document]
Comment (kevinccbsg) [reply generators/app/templates/root/_system.js:6]: Oh! copy and paste of one of my tests. Good catch 🔍

File: generators/app/templates/root/_package.json (hunk 1 above)
Comment (kevinccbsg) [reply generators/app/templates/root/_package.json]: Fix in this commit https://github.com/infinitaslearning/generator-systemic/pull/13/commits/6b883afe6fbe99240113fdabd9f252fc5ecbf95f

File: generators/app/templates/root/_system.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (kevinccbsg) [reply generators/app/templates/root/_system.js]: https://github.com/infinitaslearning/generator-systemic/pull/13/commits/6b883afe6fbe99240113fdabd9f252fc5ecbf95f

**Pull request merged**